test/util/graph_gen.h

```diff
@@ -0,0 +1,29 @@
+#include <utility>
+
+#ifndef TEST_GRAPH_GEN_H
+#define TEST_GRAPH_GEN_H
+
+typedef struct genSet {
+  long n;            // number of nodes
+  double p;          // prob of edge between nodes
+  long m;           // number of edges
+  bool graph_model;     // G(n,p) -- 0, G(n,m) -- 1
+  int max_appearances;  // the maximum number of times an edge can show up
+                            // in the stream
+  std::string out_file; // file to write stream
+  std::string cum_out_file; // file to write cum graph
+  genSet(long n, double p, long m, bool graph_model, int max_appearances,
+         std::string out_file, std::string cum_out_file)
```

Comment:
You mention elsewhere that sample.txt and cum_sample.txt are the default values for out_file and cum_out_file. It might be better to set those here to avoid needing to specify elsewhere.

Reply:
They're set as default params in `generate_stream()`. I'd like to keep them there, since technically it's a standard that we're setting, not one set by `GTgraph`